content/blog/2018-02-27-sourcebots-2018-competition.md

```diff
@@ -0,0 +1,12 @@
+---
+title: SourceBots 2018 Competition
+date: 2018-02-27
+---
+
+Today's the day we announce the date of the competition!
```

Comment:
Is it? We told them via email last week!

Reply:
Notice the fact it's backdated :wink:

Reply:
Given the delay in us shipping this PR I suggest not doing the back-dating -- we're now more than a month later.